### ISSUE TITLE
Patch with invalid "op" value returns 500

### ISSUE THREAD

**Issue by jivpif** [simpleidserver/SimpleIdServer#73]
Hello,

We found kind of a minor issue:
When we try to execute a patch with invalid "op" value (different from "add", "replace" or "remove") we get a 500 error with NullReference exception:

**Request:**
```
{
     "schemas": [
         "urn:ietf:params:scim:api:messages:2.0:PatchOp"
     ],
     "Operations": [
         {
             "op": "invalid-op",
             "path": "roles[value eq \"Team Member\"].display",
             "value": "Member"
         }
     ]
}
```

**Actual result:**
```
{
    "status": "500",
    "response": {
        "schemas": [
            "urn:ietf:params:scim:api:messages:2.0:Error"
        ],
        "status": "500",
        "scimType": "internalServerError",
        "detail": "System.NullReferenceException: Object reference not set to an instance of an object.\n   at SimpleIdServer.Scim.Commands.Handlers.PatchRepresentationCommandHandler.CheckParameter(PatchRepresentationParameter patchRepresentation)\n   at SimpleIdServer.Scim.Commands.Handlers.PatchRepresentationCommandHandler.Handle(PatchRepresentationCommand patchRepresentationCommand)\n   at SimpleIdServer.Scim.Api.BaseApiController.InternalPatch(String id, PatchRepresentationParameter patchRepresentation)"
    }
}
```

**Expected:** Status code 400 with an error.

Thanks in advance for the time.

**Comment by simpleidserver** [comment]
Hello,

Thank you for the bug report ! the issue has been fixed in the branch "release/1.2.0" and an integration test has been added to check the exception.

**Comment by jivpif** [comment]
Thanks a lot for the quick fix!
